### PR TITLE
Readme changed adding dependencies

### PR DESCRIPTION
**Pull request by Manas-kashyap**
Changes done.

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -9,4 +9,22 @@ Example command
 livewall -ov -ni -fs -- mpv -wid WID --keepaspect=no --loop PATH-OF.gif 
 
 ```
+Dependencies required :-
+
+```
+xorg-dev
```
Comment (xeon-zolt) [inline readme.md]: list would look good

File: readme.md [hunk 2]
```diff
@@ -9,4 +9,22 @@ Example command
...
+build-essential
+libx11-dev
+x11proto-xext-dev
+libxrender-dev
+libxext-dev 
+
+```
+
+Command that will do the work
```
Comment (xeon-zolt) [inline readme.md]: can you change it to cause language should be simple and clear for everyone 

> to install dependencies

**Review by xeon-zolt — CHANGES_REQUESTED** [review]

File: readme.md (hunk 1 above)
Comment (gabru-md) [reply readme.md]: Agreed @xeon-zolt

File: readme.md (hunk 2 above)
Comment (gabru-md) [reply readme.md]: Something like : `Following command will install the dependencies`

**Review by gabru-md — CHANGES_REQUESTED** [review]

**Review by xeon-zolt — APPROVED** [review]
LGTM

**Review by ParthS007 — APPROVED** [review]
LGTM !

**Pull request merged**